kwok/Dockerfile

```diff
@@ -1,15 +1,22 @@
 FROM registry.k8s.io/kwok/cluster:v0.4.0-k8s.v1.27.3
 
-ARG CLUSTER_NAME=kwok
-ENV CLUSTER_NAME=${CLUSTER_NAME}
+ENV HOST_CLUSTER=host
+ENV MEMBER_CLUSTER=m01
+ENV RH_CLUSTER=rh01
 
 COPY --chmod=755 user-signups.sh toolchain.dev.openshift.com_usersignups.yaml ./
 COPY --chmod=644 inittab /etc/inittab
 
-RUN KWOK_KUBE_APISERVER_PORT=0 kwokctl create cluster --name "$CLUSTER_NAME" || exit 1 && \
-    kwokctl --name="$CLUSTER_NAME" kubectl apply -f toolchain.dev.openshift.com_usersignups.yaml && \
-    sh ./user-signups.sh && \
-    rm user-signups.sh toolchain.dev.openshift.com_usersignups.yaml && \
-    kwokctl stop cluster --name="$CLUSTER_NAME"
+RUN KWOK_KUBE_APISERVER_PORT=0 kwokctl create cluster --name "$HOST_CLUSTER" || exit 1 && \
+    KWOK_KUBE_APISERVER_PORT=0 kwokctl create cluster --name "$MEMBER_CLUSTER" || exit 1 && \
+    KWOK_KUBE_APISERVER_PORT=0 kwokctl create cluster --name "$RH_CLUSTER" || exit 1 && \
+    kwokctl --name="$HOST_CLUSTER" kubectl apply -f toolchain.dev.openshift.com_usersignups.yaml && \
```

Comment:
Does that mean we are applying the same workspace mapping to all 3 clusters? wouldn't that make it harder to test?

Reply:
line 13 is only applied to the host cluster.

Reply:
I referred to the fact all 3 lines use the same file (`toolchain.dev.openshift.com_usersignups.yaml`)

Reply:
In what sense are they using the same file?

Reply:
Not sure if that is ok or not.

Reply:
Maybe I'm misunderstanding the 'code', but all of them use `apply -f toolchain.dev.openshift.com_usersignups.yaml`